metadata/utils.go

```diff
@@ -37,6 +37,12 @@ const (
 
 	// QueryIndexName defines the metadata key for the name of query indexing schema (for redis).
 	QueryIndexName = "queryIndexName"
+
+	// MaxBulkCountKey is the key for the max bulk count in the metadata.
+	MaxBulkCountKey string = "maxBulkCount"
```

Comment:
Is this an attribute proposed for all components? If this is Kafka specific, just keep it in kafka.go.

Reply:
Yes, ideally "maxBulkCount" and "maxBulkAwaitDurationMilliSeconds" are attributes proposed for all components. As, while consuming messages from broker, there needs to be some limit, either in terms of count of messages or max await time, after which BulkSubscribe handler to send ahead messages should be called.